app/src/tamilNadu/res/drawable/thanks.xml

```diff
@@ -0,0 +1,24 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Let's combine this and definition illustration into a single drawable. I have combine them on Figma, so you should be able to export them properly now.

We are using `ic_` for icons or `illustration_` for illustrations for all the new drawables we are adding. So, let's rename the drawable

Reply:
That should hopefully help with the long path datas as well for these illustrations as well a bit